lib/fastlane/actions/bundle_install.rb

```diff
@@ -34,8 +34,19 @@ def self.run(params)
       # rubocop:enable Metrics/CyclomaticComplexity
       # rubocop:enable Metrics/PerceivedComplexity
 
+      def self.gemfile_exists?(params)
+        possible_gemfiles = ['Gemfile', 'gemfile']
+        possible_gemfiles.insert(0, params[:gemfile]) if params[:gemfile]
```

Comment:
```<<``` does work, but it pushes onto the array making the user provided gemfile the last in the list to look for. This would then echo a message to the user saying that it could not find Gemfile or gemfile before continuing on to finding the user provided one.

I could write this as:

```ruby
possible_gemfiles = []
possible_gemfiles << params[:gemfile] if params[:gemfile]
possible_gemfiles << 'Gemfile' << 'gemfile'
```